merino/curated_recommendations/protocol.py

```diff
@@ -308,3 +381,22 @@ class CuratedRecommendationsResponse(BaseModel):
     data: list[CuratedRecommendation]
     feeds: dict[str, Section] | None = None
     interestPicker: InterestPicker | None = None
+
+
+class CuratedRecommendationsLegacyResponse(BaseModel):
+    """Response schema for a list of curated recommendations for
+    the /curated-recommendations/legacy-115-129 endpoint
+    """
+
+    data: list[CuratedRecommendationLegacy]
+
+
+class CuratedRecommendationsGlobalLegacyResponse(BaseModel):
```

Comment:
I would avoid using 'global' in the name, because a future dev could think that this endpoint serves recommendations globally, and the other legacy response does not. I would consistently use the Firefox versions to differentiate these two endpoints and their corresponding classes.

I like how you've named the endpoints, and that you refer to it in the following comment!
```suggestion
class CuratedRecommendationsLegacyFx114Response(BaseModel):
```